packages/devkit/src/generators/target-defaults-utils.spec.ts

```diff
@@ -129,14 +124,9 @@ describe('target-defaults-utils', () => {
           ],
         }
       `);
-      expect(newNxJson.targetDefaults['cypress:e2e-ci--**/*'])
-        .toMatchInlineSnapshot(`
-        {
-          "dependsOn": [
-            "^build-base",
-          ],
-        }
-      `);
+      expect(
+        newNxJson.targetDefaults['cypress:e2e-ci--**/*']
+      ).toMatchInlineSnapshot(`undefined`);
     });
 
     it('should not add additional e2e-ci--**/* target default for e2e plugin when it already exists with build target', async () => {
```

Comment:
It looks like this test is meant to check that if a target default for `e2e-ci--**/**` already exists, it shouldn't be duplicated. We need to replace all the `e2e-ci--**/*` with `e2e-ci--**/**`
```suggestion
    it('should not add additional e2e-ci--**/** target default for e2e plugin when it already exists with build target', async () => {
```